external/rdkit_inchi_isascii.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/External/INCHI-API/CMakeLists.txt b/External/INCHI-API/CMakeLists.txt
```

Comment:
I asked this before, but now I don't remember -- should we be patching RDKit directly with this?